resources/l10n/bg/LC_MESSAGES/loot.po

```diff
@@ -552,91 +618,107 @@ msgstr ""
 "„Skyrim“, „Enderal“, „Skyrim SE“, „Enderal SE“, „Skyrim VR“, „Fallout 3“, "
 "„Fallout: NV“, „Fallout 4“ и „Fallout 4 VR“."
 
-#: src/gui/qt/main_window.cpp:1838
+#: src/gui/qt/main_window.cpp:1903
 msgid "About LOOT"
 msgstr "За „LOOT“"
 
-#: src/gui/qt/main_window.cpp:2142
+#: src/gui/qt/main_window.cpp:2213
 msgid "Identifying conflicting plugins..."
 msgstr "Откриване на несъвместимите приставки..."
 
-#: src/gui/qt/main_window.cpp:2285
+#: src/gui/qt/main_window.cpp:2358
 msgid ""
 "Auto-sort has been cancelled as there is at least one error message "
 "displayed."
 msgstr ""
 "Автоматичното сортиране е прекратено, поради поне едно съобщение за грешка."
 
-#: src/gui/qt/main_window.cpp:2336
+#: src/gui/qt/main_window.cpp:2413
 msgid "No masterlist update was necessary."
 msgstr "Не беше необходимо обновяване на основния списък."
 
-#: src/gui/qt/main_window.cpp:2349
+#: src/gui/qt/main_window.cpp:2433
 msgid "Masterlist updated to revision {0}."
 msgstr "Основният списък е обновен до ревизия {0}."
 
-#: src/gui/qt/main_window.cpp:2407
+#: src/gui/qt/main_window.cpp:2490
+msgid "No masterlist updates were necessary."
+msgstr "Не беше необходимо обновяване на основните списъци."
+
+#: src/gui/qt/main_window.cpp:2510
+msgid ""
+"Masterlists updated for the following games:\n"
+"\n"
+msgstr ""
+"Основните списъци на следните игри са обновени:\n"
+"\n"
+
+#: src/gui/qt/main_window.cpp:2516
+msgid "LOOT"
+msgstr "„LOOT“"
```

Comment:
This doesn't need quoted, it doesn't really need a translation, so it should just be an identical string.

Reply:
Fixed.